jerry-core/api/jerry.c

```diff
@@ -4602,6 +4602,39 @@ jerry_set_realm (jerry_value_t realm_value) /**< jerry api value */
 #endif /* ENABLED (JERRY_BUILTIN_REALMS) */
 } /* jerry_set_realm */
 
+/**
+ * Gets the 'this' binding of a realm
+ *
+ * @return thrown error - if any error is happened
+ *         this value - otherwise
+ */
+jerry_value_t
+jerry_realm_get_this (jerry_value_t realm_value) /**< realm value */
+{
+  jerry_assert_api_available ();
+
+#if ENABLED (JERRY_BUILTIN_REALMS)
+  if (ecma_is_value_object (realm_value))
+  {
+    ecma_object_t *object_p = ecma_get_object_from_value (realm_value);
+
+    if (ecma_get_object_is_builtin (object_p)
+        && ecma_builtin_is_global (object_p))
```

Comment:
AFAIK, this could be a single line.

Reply:
This is the same for every other function. Shall I change them all?